inst/include/dust/interface.hpp

```diff
@@ -41,6 +41,8 @@ cpp11::sexp create_matrix(size_t nrow, size_t ncol, T& pars);
 template <typename T>
 cpp11::sexp create_array(const std::vector<size_t>& dim, T& pars);
 
+inline void check_pars_multi(cpp11::list pars, size_t n);
```

Comment:
inline 👀 
(I approve)

Reply:
has to be inline to avoid multiple definition errors :)